src/lib/list/selection-list.spec.ts

```diff
@@ -1109,6 +1111,20 @@ describe('MatSelectionList with forms', () => {
       expect(fixture.componentInstance.selectedOptions).toEqual(['opt1', 'opt2']);
     }));
 
+    it('should show the item as selected when preselected via form control', fakeAsync(() => {
+         const fixture = TestBed.createComponent(SelectionListWithPreselectedFormControl);
+         fixture.detectChanges();
+
+         const option = fixture.debugElement.queryAll(By.directive(MatListOption))[1];
+
+         fixture.detectChanges();
+         flush();
+         fixture.detectChanges();
+
+         expect(option.componentInstance.selected).toBe(true);
+         expect(option.nativeElement.querySelector('.mat-pseudo-checkbox-checked')).toBeTruthy();
```

Comment:
The indentation seems off here. isn't clang-format configured to use two spaces? If so, not something you can fix here.

Reply:
I hadn't run it through clang. I did now, but the entire file ended up getting reformatted.